charts/hlf-k8s/templates/deployment-system-channel-operator.yaml

```diff
@@ -30,87 +30,87 @@ spec:
           - |
             ## Update CA certs
             update-ca-certificates
+
             ## Check connection with the Orderer
             printf "[DEBUG] Testing the connection with the Orderer ({{ .Values.orderer.host }}:{{ .Values.orderer.port }})\n"
             until $(nc -z {{ .Values.orderer.host }} {{ .Values.orderer.port }}); do
               printf "[DEBUG] Orderer ({{ .Values.orderer.host }}:{{ .Values.orderer.port }}) is not reacheable, retry in 5s\n"
               sleep 5
             done
 
-            ## Add orgs to system channel
-
+            ## Add orgs to the system channel
             while true; do
 
-              printf "[DEBUG] Creating system channel ({{ .Values.systemChannel.name }}) updates\n"
+              printf "[DEBUG] Creating updates for system channel '{{ .Values.systemChannel.name }}'\n"
 
-              while IFS=" " read -r orgName configUrl ; do
+              while IFS=" " read -r org configUrl ; do
```

Comment:
`orgName` is used here because we have it in https://github.com/SubstraFoundation/hlf-k8s/pull/45/files#diff-64116e958a242f7cd14863d97fdb5c77R40 values.
To simplify the understanding where variables are use we should use same name. Can you change it here or in the values ?